www/http/server.go

```diff
@@ -72,6 +74,7 @@ func (s *Server) StartServer(grpcServer string) error {
 	s.router.HandleFunc("/account/number/{number}", s.GetAccountByNumberHandler)
 	s.router.HandleFunc("/validator/address/{address}", s.GetValidatorHandler)
 	s.router.HandleFunc("/validator/number/{number}", s.GetValidatorByNumberHandler)
+	http.Handle("/metrics/prometheus", promhttp.Handler())
```

Comment:
@b00f 
unfortunately, we can't use this because `HandleFunc` returns `(w http.ResponseWriter, r *http.Request)` but `promhttp.Handler()` returns `http.Handler`